scripts/run-external-tests.js

```diff
@@ -20,6 +20,10 @@ function tryFormat(file) {
   return null;
 }
 
+/**
+ *
```

Comment:
Can you delete this line?

Reply:
You mean it should be
```js
/**
 * @param {string | string[]} patterns
 */
```

or

```js
/** @param {string | string[]} patterns */
```

How do you want it?

Reply:
```ts
/**
 * @param {string | string[]} patterns
 */
```